.github/workflows/conda-python-build.yaml

```diff
@@ -73,6 +77,7 @@ jobs:
       fail-fast: false
       matrix: ${{ fromJSON(needs.compute-matrix.outputs.MATRIX) }}
     runs-on: "linux-${{ matrix.ARCH }}-${{ inputs.node_type }}"
+    continue-on-error: ${{ matrix.PY_VER == '3.11' }}
```

Comment:
What is the purpose of these `continue-on-error` directives?

Reply:
I'll explain this note from the description in more detail:

> *I'm proposing using GitHub's continue-on-error ([docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idcontinue-on-error)) to allow the Python 3.11 jobs to fail on PRs without blocking those PRs from being merged.*

Doing this allows us to merge this PR, add support one RAPIDS repo at a time, and then come back and remove this to make those required again.

If we just merged this PR as-is without `continue-on-error`, CI would be broken in any RAPIDS repos not yet supporting Python 3.11 (which is most of them right now) until they were fixed. That'd be really disruptive to development there.

Another alternative is to not do this `continue-on-error` thing, and instead do 2 rounds of PRs to all the RAPIDS repos:

* round 1: updating all their GitHub Actions configs to like `{workflow}.yaml@python-3.11`, getting all those building successfully, then merging them all in quick succession
* round 2: merging this PR, then updating all those configs in the repos again to point at `{workflow}.yaml@branch-24.04`

That's what we did for CUDA 12.2 support (https://github.com/rapidsai/build-planning/issues/7) and it increased the amount of effort required for everyone involved. Hoping to avoid that here to make this update go smoother and faster.

Reply:
Sounds like a good plan!

Reply:
That being said, might I suggest adding a TODO comment referencing the 3.11 issue? Something people can reference to understand why this directive is here.

Reply:
I'm really not a fan of TODO comments in code, I've found that they tend to be forgotten more easily than issues in a backlog. There's already a checklist at the top of this PR that includes a step "make these jobs required", could that be enough?

Reply:
I absolutely agree we should have the issue in a backlog. I was thinking we could *also* have a TODO comment that references said issue. WDYT?

Reply:
Ok sure, no problem.

Added in https://github.com/rapidsai/shared-workflows/pull/176/commits/c32c724b8f2ed0b29e9b73c7661262094862456c

Reply:
I see the comment in `wheels-build.yaml` and `wheels-test.yaml`. Please add it to `conda-python-build.yaml` and `conda-python-tests.yaml` too.

Reply:
https://blog.stackademic.com/vscode-auto-save-22b6ce1c81e1

Reply:
thanks @msarahan ! added!